code-challenges.js

```diff
@@ -32,8 +67,102 @@ var hand4 = [3, 5, 3, 5, 3]
 // Expected output: true
 
 
+// //this function will map over the array and check the values at each index. we will need to have the values occur either 2 or 3 times 
+// //we can set variables for the numbers that will make up the array of values.
+// // value1 != value2
+// // value1 occurs 3 times
+// // value2 occurs 2 times
+// //may need to set initial starting counts for value1 & value2 and then count up from there
+
+// const fullHouse = (array) => {
```

Comment:
good comments on explaining your strategy! Consider using a data structure that can model a full house.